node/env0-deploy-utils.js

```diff
@@ -7,9 +7,9 @@ class DeployUtils {
     await apiClient.init(options.apiKey, options.apiSecret);
   }
 
-  async getEnvironment(environmentName, organizationId) {
-    console.log(`getting all environments organizationId: ${organizationId}`);
-    const environments = await apiClient.callApi('get', `environments?organization=${organizationId}`);
+  async getEnvironment(environmentName, projectId) {
+    console.log(`getting all environments organizationId: ${projectId}`);
+    const environments = await apiClient.callApi('get', `environments?organization=${projectId}`);
```

Comment:
don't you need to change the querystring key as well (`organization`) ?

Reply:
maybe change the log as well

Reply:
Right. [Done](https://github.com/env0/env0-client-integrations/pull/3/commits/88d870cfdb7e4a564643cf1f73f49bdd76b0887e)